src/environment/App/App.jsx

```diff
@@ -7,16 +8,60 @@
 
 /* eslint-disable react/require-default-props */
 // eslint-disable-next-line no-use-before-define
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import {ShopperCustomers, ShopperSearch} from 'lib';
+import { ShopperLogin, ShopperSearch } from 'lib';
 import config from '../config';
 import './App.css';
 
-const customerClient = new ShopperCustomers(config);
+// For this client, we want to explicitly handle redirects.
+const loginClient = new ShopperLogin({
+  fetchOptions: { redirect: 'manual' },
+  ...config
+})
```

Comment:
1️⃣  – here we create a client and set our intent for it to not follow redirects automajically.